src/heretic/g_game.c

```diff
@@ -1775,7 +1782,28 @@ void G_RecordDemo(skill_t skill, int numplayers, int episode, int map,
     *demo_p++ = episode;
     *demo_p++ = map;
 
-    for (i = 0; i < MAXPLAYERS; i++)
+    // Write special parameter bits onto player one byte.
+    // This aligns with vvHeretic demo usage:
+    //   0x20 = -respawn
+    //   0x10 = -longtics
+    //   0x02 = -nomonsters
+
+    *demo_p = 1; // assume player one exists
+    if (respawnparm)
+    {
+        *demo_p += 32;
+    }
+    if (longtics)
```

Comment:
I think all of these should be longtics-only, correct? Vanilla won't understand these. Or is it harmless?